core/src/main/resources/lib/layout/copyButton/copyButton.js

```diff
@@ -4,31 +4,20 @@ Behaviour.specify(
   0,
   function (copyButton) {
     copyButton.addEventListener("click", () => {
-      // HTMLUnit 2.70.0 does not recognize isSecureContext
-      // https://issues.jenkins.io/browse/JENKINS-70895
-      if (!window.isRunAsTest && isSecureContext) {
-        // Make an invisible textarea element containing the text
-        const fakeInput = document.createElement("textarea");
-        fakeInput.value = copyButton.getAttribute("text");
-        fakeInput.style.width = "1px";
-        fakeInput.style.height = "1px";
-        fakeInput.style.border = "none";
-        fakeInput.style.padding = "0px";
-        fakeInput.style.position = "absolute";
-        fakeInput.style.top = "-99999px";
-        fakeInput.style.left = "-99999px";
-        fakeInput.setAttribute("tabindex", "-1");
-        document.body.appendChild(fakeInput);
-
-        // Select the text and copy it to the clipboard
-        fakeInput.select();
-        navigator.clipboard.writeText(fakeInput.value);
-
-        // Remove the textarea element
-        document.body.removeChild(fakeInput);
-
-        // Show the completion message
-        hoverNotification(copyButton.getAttribute("message"), copyButton);
+      if (isSecureContext) {
+        // Copy the text to the clipboard
+        navigator.clipboard
```

Comment:
whilst mainstream browsers support this (according to canIUse) I believe unless you set a handler in `HtmlUnit` that this will be undefined when runnign tests with webclient.

the [webclient created in j-t-h](https://github.com/jenkinsci/jenkins-test-harness/blob/master/src/main/java/org/jvnet/hudson/test/JenkinsRule.java#L2329-L2333) is not configured with a handler.